methods_and_flow_control/string_practice.rb

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
+
+string_practice = "this is a string to practice with"
+practice = "practice "
+the_string = "The string '"
+length_of_characters = "' has 33 characters"
```

Comment:
Isn't there a method to figure out how long a string is?  No need to hard code it...